shared/src/commonTest/kotlin/co/nimblehq/avishek/phong/kmmic/helper/MockUtil.kt

```diff
@@ -14,6 +18,45 @@ object MockUtil {
         "coverImageUrl"
     )
 
+    val mockSurveys = listOf<Survey>(
+        Survey(
+            "id-1",
+            "title",
+            "description",
+            true,
+            "coverImageUrl"
+        ),
+        Survey(
+            "id-2",
+            "title",
+            "description",
+            true,
+            "coverImageUrl"
+        ),
+        Survey(
+            "id-3",
+            "title",
+            "description",
+            true,
+            "coverImageUrl"
+        ),
+        Survey(
+            "id-4",
+            "title",
+            "description",
+            true,
+            "coverImageUrl"
+        ),
+        Survey(
+            "id-5",
+            "title",
+            "description",
+            true,
+            "coverImageUrl"
+        )
+
+    )
```

Comment:
We can do this here:
```suggestion
    val mockSurveys = listOf<Survey>(
        mockSurvey.copy(id = id-1),
        mockSurvey.copy(id = id-2),
        mockSurvey.copy(id = id-3),
        mockSurvey.copy(id = id-4),
        mockSurvey.copy(id = id-5),
    )
```